### DISCUSSION TITLE
Can you please guide me to find possible issue with my bevy_tiled_toolkit crate

### DISCUSSION THREAD

**Discussion by 39george** [StarArawn/bevy_ecs_tilemap#452] [Q&A] [answered]
Hello, I'm making a crate for comfortable working with tiled editor: [bevy_tiled_toolkit](https://github.com/ghashy/bevy_tiled_toolkit).
`bevy_tiled_toolkit` has an opportunity to render a layer with your crate, `bevy_ecs_tilemap`, but there is a problem with respawning layer. This problem occurs only in my crate, in you examples it does not exist.
I spawn a layer with `bevy_ecs_tilemap` and store layer's entity in `layer_storage` among other layer's entities (they can be spawned not in `bevy_ecs_tilemap`), then I push all layers to the entire map's children to create an hierarchy.
Then when I despawn all layers and map, it looks correctly.
After that I spawn the entire map again, and layer, spawned with `bevy_ecs_tilemap` is not visible, although `bevy-inspector-egui` shows me that every thing was spawned correctly, all textures are loaded and transforms are right, and tiles should be visible, but I can't see it. 
Can you please help me what could cause the problem?

https://github.com/StarArawn/bevy_ecs_tilemap/assets/109857267/0105a5cc-4488-4c67-93f7-fb1a3fe30b91

**Comment by rparrett** [comment, marked as answer]
It sounds like you are describing #450 which was fixed in the main branch but not included in a release yet. Could you try using the main branch and see if that solves the issue?

**Reply by 39george** [reply]
Thank you very much! It solves the problem, I tried to make it work for two days, and now I am totally satisfied :)
